tests/rptest/clients/admin/proto/redpanda/core/admin/v2/cluster_pb2.pyi

```diff
@@ -34,7 +34,7 @@ class ListKafkaConnectionsRequest(google.protobuf.message.Message):
     FILTER_FIELD_NUMBER: builtins.int
     ORDER_BY_FIELD_NUMBER: builtins.int
     page_size: builtins.int
-    'The maximum number of connections to return. If unspecified or 0, a\n    default value may be applied. Note that paging is currently not fully\n    supported, and this field only acts as a limit for the first page of data\n    returned. Subsequent pages of data cannot be requested.\n    '
+    'The maximum number of connections to return. If unspecified or 0, a\n    default value may be applied. The server may return fewer connections\n    than requested due to memory constraints; the limit is set to allow\n    listing all connections for a single broker. Consider filtering by\n    node_id to view connections for specific brokers. Note that paging is\n    currently not fully supported, and this field only acts as a limit for\n    the first page of data returned. Subsequent pages of data cannot be\n    requested.\n    '
```

Comment:
I'm guessing someone forgot to run `task rp:generate-ducktape-protos` after modifying the cluster proto at some point. Happy to separate this into a separate commit if folks want.

Reply:
Should we add a note to `proto/redpanda/README.md` about running the command after updating a proto?

Reply:
I will just add a github action to check in another PR